draft-ietf-webtrans-http2.md

```diff
@@ -257,6 +257,11 @@ the asynchronous nature of the protocol; instead, it MUST reply to the CONNECT
 request with a status code 429, indicating that the client attempted to open
 too many sessions.
 
+Any values in subsequent SETTINGS frames sent by either endpoint MUST NOT be
+lower than a previously sent value; an endpoint that receives a SETTINGS frame
+with a lowered value MUST close the connection with a connection error of type
+PROTOCOL_ERROR ({{Section 5.4.1 of HTTP2}}).
```

Comment:
HTTP/2 doesn't really have a problem with limits being reduced, so I'm not sure that we need this one.

Reply:
I guess instead of prohibiting this, we can pull the same text from H2: 

> An endpoint that wishes to reduce the value of <SETTING_NAME> to a value that is below the current number of open streams can either close streams that exceed the new value or allow streams to complete.

Reply:
Updated with this for both